main/admin/course_add.php

```diff
@@ -47,7 +47,7 @@
     get_lang('Title'),
     true,
     array(
-        'id' => 'title'
+        'aria-label' => get_lang('Title')
```

Comment:
Now, in some cases, is not necessary add an id or aria-label (please update your repo)